modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/assignment/MultiLanguageKaleoTaskAssignmentSelector.java

```diff
@@ -60,7 +60,7 @@ public Collection<KaleoTaskAssignment> getKaleoTaskAssignments(
 		KaleoTaskAssignmentSelector kaleoTaskAssignmentSelector = null;
 
 		if (assigneeClassName.equals(ResourceAction.class.getName())) {
-			kaleoTaskAssignmentSelector = _kaleoTaskAssignmentSelectors.get(
+			kaleoTaskAssignmentSelector = _serviceTrackerMap.getService(
```

Comment:
@W2jx The code before your change does not seem right, how could you retrieve something by "assigneeClassName" when the key is script language.